README.md

```diff
@@ -316,6 +317,10 @@ docker run -p 8080:80 \
     bgruening/galaxy-stable
 ```
 
+## On-demand reference data with CVMFS <a name="cvmfs" /> [[toc]](#toc)
+By default, Galaxy instances launched with this image will have on-demand access to approximately 3TB of reference genomes and indexes. These are the same reference data available on the main Galaxy server. This is achieved by connecting to Galaxy's CernVM filesystem (CVMFS) server at `data.galaxyproject.org`.
+
```

Comment:
We need to start Galaxy with --privileged isn't it to use this feature, isn't it?
Maybe we can mention that CVMFS is a cached file system, so that data is only transferred if it is really accessed and than cached locally.